examples/push_pull_inherit.py

```diff
@@ -5,7 +5,7 @@
 
 
 class Greeter(Agent):
-    def on_init(self):
+    def configure(self):
         self.bind('PUSH', alias='main')
```

Comment:
I would revert this. `configured` is not yet recommended over `on_init` (they both serve for different purposes).